pyiron/atomistics/structure/atoms.py

```diff
@@ -331,6 +330,51 @@ def set_high_symmetry_points(self, new_high_symmetry_points):
             raise ValueError("has to be dict!")
         self._high_symmetry_points = new_high_symmetry_points
 
+    def add_high_symmetry_points(self, new_points):
+        """
+        Adds new points to the dict of existing high symmetry points.
+
+        Args:
+            new_points (dict): Points to add
+        """
+        if self.get_high_symmetry_points() is None:
+            raise AssertionError("Construct high symmetry points first. Use self.band_structure_calculations().")
+        else:
+            self._high_symmetry_points.update(new_points)
+
+    def get_high_symmetry_path(self):
+        """
+        Path used for band structure calculations
+
+        Returns:
+            dict: dict of pathes with start and end points.
+
+        """
+        return self._high_symmetry_path
+
+    def _set_high_symmetry_path(self, new_path):
+        """
+        Sets new list for the high symmetry path used for band structure calculations.
+
+        Args:
+            new_path (dict): dictionary of lists of tuples with start and end point.
+                E.G. {"my_path": [('Gamma', 'X'), ('X', 'Y')]}
+        """
+        self._high_symmetry_path = new_path
+
+    def add_high_symmetry_path(self, path):
+        """
+        Adds a new path to the dictionary of pathes for band structure calculations.
+
+        Args:
+            path (dict): dictionary of lists of tuples with start and end point.
+                E.G. {"my_path": [('Gamma', 'X'), ('X', 'Y')]}
+        """
+        if self.get_high_symmetry_path() is None:
+            raise AssertionError("Construct high symmetry path first. Use self.band_structure_calculations().")
```

Comment:
`self.band_structure_calculation` and not `self.band_structure_calculations`

Reply:
done